src/utils/articleTextSize.js

```diff
@@ -1,8 +1,11 @@
+/**
+ * The font size state is set to the HTML body to be made available
+ * for all elements with adjustable text size. See fontsize.less for more info
+ */
+
 const KEY = 'article-textsize'
```

Comment:
The storage key is unchanged but the values it can contain have changed from [0,6] to [1,7].

How would the new code react when it reads the value 0? 

I see a few ways of handling that:
1. On read, validate the value and migrate it (0 becomes 1)
2. Version the key (KEY = 'article-textsize-v2')
3. Keep the stored variable domain as [0,6] and add 1 when you build the class so that everybody's setting is retained.

Reply:
Hmm good point, all of these ideas are fair. Sort of leaning towards the third option... let me try it out